### ISSUE TITLE
Missing geo data in lobid GND data: blank nodes lost during conversion from ntriples to json-ld

### ISSUE THREAD

**Issue by dr0i** [lobid/lodmill#503]
In the GND reside some geo data, see e.g. http://d-nb.info/gnd/4074335-4.
In lobid this data is missing , see e.g. http://lobid.org/subject?id=4074335-4, also the data is correctly loaded into hdfs - try:

```
hduser@weywot1:~$ hadoop fs -cat extlod/gnd/407.nt | grep 4074335-4 | grep hasGe
```

This yields the triple:

```
<http://d-nb.info/gnd/4074335-4> <http://www.opengis.net/ont/geosparql#hasGeometry> _:BX2D7baa6ceeX3A1464867c069X3A2440 .
```

After the conversion to json-ld, thsi triple is missing

```
hadoop fs -cat output/json-ld-gnd/* | grep '"@value":"(DE-588c)4074335-4' | grep hasGe
```

It seems that blank nodes are lost when ntriples are converted to json-ld.

**Comment by acka47** [comment]
Werner Greßhof reported multiple times that he would be happy to have the geodata in GND.

**Comment by acka47** [comment]
This will be fixed in 2.0, wonÄt be fixed in 1.0. Closing.
